pkg/skaffold/build/cache/retrieve.go

```diff
@@ -138,7 +138,7 @@ func (c *cache) Build(ctx context.Context, out io.Writer, tags tag.ImageTags, ar
 		})
 	}
 
-	log.Entry(ctx).Info("Cache check completed in", util.ShowHumanizeTime(time.Since(start)))
+	log.Entry(ctx).Infoln("Cache check completed in", util.ShowHumanizeTime(time.Since(start)))
```

Comment:
nit: Are there any integration/unit tests that verify the form of a log entries?

Reply:
We currently don't really have any testing for most things output related I believe. This would be good to have in the future and I think that having the API Server now will help a lot with that since we can connect to the server and check output through there rather than parsing output from the terminal